### PR TITLE
Current law parameter updates

### PR DESCRIPTION
**Pull request by andersonfrailey**
This PR updates `current_law_policy.json` to reflect the latest IRS release. It address issue #1165 opened by @MattHJensen. All updates came from [this](https://www.irs.gov/uac/newsroom/in-2017-some-tax-benefits-increase-slightly-due-to-inflation-adjustments-others-are-unchanged) IRS website and the [revenue procedure](https://www.irs.gov/pub/irs-drop/rp-16-55.pdf) document linked there, except for social security maximum table income, which can be found [here](https://www.ssa.gov/policy/docs/quickfacts/prog_highlights/RatesLimits2017.html).

The new parameters resulted in changes in `pufcsv_agg_expect.txt` and `pufcsv_mtr_expect.txt`. `test_policy.csv` was also updated so that the changes would not cause test failures.

### CODE REVIEW

**Comment by MattHJensen** [comment]
I am reviewing this now. 

cc @PeterDSteinberg @brittainhard

File: taxcalc/current_law_policy.json [hunk 1]
```diff
@@ -2293,13 +2363,15 @@
         "row_label": ["2013",
                       "2014",
                       "2015",
-                      "2016"],
+                      "2016",
+                      "2017"],
         "cpi_inflated": false,
```
Comment (MattHJensen) [inline taxcalc/current_law_policy.json]: @andersonfrailey, do you think this should be `'cpi_inflated': true` ?

**Comment by martinholmer** [comment]
@codykallen,  Are the [new Growth.REAL_GDP_GROWTH_RATES](https://github.com/martinholmer/Tax-Calculator/blob/a94bfc80e2be1b01c6e07bc04e828f92eb31dbe7/taxcalc/growth.py#L45-L47) (when converted to percentage terms and rounded to one decimal point) similar to those in any recent CBO 10-year economic projections?

**Comment by MattHJensen** [comment]
> @codykallen, Are the new Growth.REAL_GDP_GROWTH_RATES (when converted to percentage terms and rounded to one decimal point) similar to those in any recent CBO 10-year economic projections?

@martinholmer, was this meant for a different issue?

**Comment by MattHJensen** [comment]
@andersonfrailey, beyond my one comment this looks good to me. I'll merge when you give me the green light.

File: taxcalc/current_law_policy.json (hunk 1 above)
Comment (andersonfrailey) [inline taxcalc/current_law_policy.json]: @MattHJensen, yes. I found this line in a [CBO article](https://www.cbo.gov/budget-options/2016/52264) that indicates this should be `'cpu_inflated': true`.
>This option would lower the threshold for the EITC investment income test from $3,400 to $1,700. As under current law, that threshold would be indexed to include the effects of inflation.

I'll make the change and submit a new commit.

**Comment by andersonfrailey** [comment]
Forgot to re-run the tests before latest commit and there are slight changes to `pufcsv_agg_expect.txt`. So ignore this commit and I'll get it fixed.

**Comment by andersonfrailey** [comment]
@MattHJensen ready to merge.

**Comment by MattHJensen** [comment]
Thanks @andersonfrailey. Merging.

**Comment by martinholmer** [comment]
@MattHJensen said in Tax-Calculator issue #1169:
> @Peter-Metz agreed to take on this issue but asked for additional explanation for what needs to be done.
> ...

@Peter-Metz's help would be appreciated, but it makes no sense to begin the coding part of this task until Tax-Calculator is updated with the new `taxdata` information based on the newest CBO projections.  That updated information, which includes the inflation rates need for this task, is contained in pending [`taxdata` pull request 295](https://github.com/PSLmodels/taxdata/pull/295#issuecomment-451589159), which has serious problems that need to be resolved.

What @Peter-Metz could start working on now (while taxdata 295 is fixed and while I incorporate the fixed projection information into Tax-Calculator) would be the critical task of gathering the PDF files containing the **final** (not draft) 2018 IRS forms (and instructions).   These are the key documents needed to specify the actual (as opposed to projected) values of the 2018 tax policy parameters.

**Pull request merged**